providers/dns/regru/internal/model.go

```diff
@@ -0,0 +1,85 @@
+package internal
+
+import "fmt"
+
+const successResult = "success"
+
+// APIResponse is the representation of an API response.
+type APIResponse struct {
+	Result string `json:"result"`
+
+	Answer *Answer `json:"answer,omitempty"`
+
+	ErrorCode   string            `json:"error_code,omitempty"`
+	ErrorParams map[string]string `json:"error_params,omitempty"`
```

Comment:
`ErrorParams` seems to be unused.

Reply:
The `ErrorsParams` contaings not really useful information:

```json
{
   "charset" : "utf-8",
   "error_code" : "NO_AUTH",
   "error_params" : {
      "command_name" : "zone/remove_record"
   },
   "error_text" : "No authorization mechanism selected",
   "messagestore" : null,
   "result" : "error"
}
```

So I think we will ignore it for now.

Reply:
Yeah, fine with me.